### PR TITLE
ci: change behavior for CI flow

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR changes the behavior of the CI flow.  In order to push the changes to production, a comment with the following value is required `/test librarium_release_build`. The comment in the PR triggers the build step for production. Now that preview URLs are available, we no longer need the `docs-latests.spectrocloud.com` URL and flow. We want to merge into the base branch, `master`, to automatically deploy to production.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1008#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **zulfilee**
You can assign the PR to them by writing `/assign @zulfilee` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/master/prow/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | e2a70dce43f4632a81f437d059b8dec907a0f3e9 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1008/librarium_build/1597835090568679424) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by zulfilee — APPROVED** [review]
Looks good. PR merged will now automatically push the change to docs.spectrocloud.com

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1008#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1008#pullrequestreview-1202663609" title="Approved">zulfilee</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/master/prow/OWNERS)~~ [zulfilee]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
